app/definitions/IMessage.ts

```diff
@@ -54,8 +59,10 @@ export interface ILastMessage {
 }
 
 export interface IMessage {
+	_id: string;
+	rid: string;
 	msg?: string;
-	t?: SubscriptionType;
+	t?: MessageType;
```

Comment:
This change is done between IMessage, IThread and IThreadMessage; not actually sure about it, but `toDecrypt` complains a lot with this.

Reply:
I'm not sure about, the difference between `SubscriptionType` and `MessageType` is huge.
Can you debug and try find out if is ok change this ?